src/utils/marketsAndNetworksConfig.ts

```diff
@@ -150,11 +150,11 @@ const providers: { [network: string]: ethersProviders.Provider } = {};
  * @param chainId
  * @returns provider or fallbackprovider in case multiple rpcs are configured
  */
-export const getProvider = (chainId: ChainId): ethersProviders.Provider => {
+export const getProvider = (chainId: ChainId, ens?: boolean): ethersProviders.Provider => {
```

Comment:
I'm wondering if it'd make sense to just have a new function, `getENSProvider`, instead of adding an optional param to this one. Seems like it's enough of a special case that it might make sense, wouldn't have to have a param for the chainId, just hardcode mainnet and return the public rpc.

Reply:
Makes sense, yeah that would be cleaner